### PR TITLE
CFY-5326 fix some log texts

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

**Comment by nir0s** [comment]
To the reviewer.

* Import ordering has changed in all modules touched by this PR. There have been some removals where necessary like the `messages.py` module which is no longer necessary and duplication of imports.
* The log message depicting the beginning of every process/action now ends with `...`
* The log message depicting the ending of every process/action now ends with nothing.
* Commands found within log message (e.g. `cfy use`...) are now enclosed by backticks.
* Please verify formatting consistency across all modules.
* Previously, (inconsistently) SOME messages printed the Manager=IP in context. `cloudify_cli/utils.py` now has a function called `manager_msg` which adds `[Manager=MANAGER_IP]` to a message if used. This is currently unused. We should decide whether and if so, where to use it.

File: cloudify_cli/commands/blueprints.py [hunk 1]
```diff
@@ -160,7 +150,7 @@ def trim_description(blueprint):
                       'created_at', 'updated_at'],
                      data=blueprints)
 
-    print_table('Blueprints:', pt)
+    utils.print_table('Availble blueprints:', pt)
```
Comment (EarthmanT) [inline cloudify_cli/commands/blueprints.py]: available

File: cloudify_cli/commands/groups.py [hunk 2]
```diff
@@ -47,7 +46,7 @@ def ls(deployment_id):
         logger.info('No groups defined for deployment {0}'.format(
             deployment.id))
     else:
-        logger.info("Deployment '{0}' groups:".format(deployment.id))
+        logger.info("Groups:".format(deployment.id))
```
Comment (EarthmanT) [inline cloudify_cli/commands/groups.py]: `python
"Groups: {0}".format(deployment.id)
`

File: cloudify_cli/commands/snapshots.py [hunk 3]
```diff
@@ -25,64 +25,57 @@
...
     logger = get_logger()
     management_ip = utils.get_management_server_ip()
-    logger.info("Restoring snapshot '{0}' at management server {1}"
-                .format(snapshot_id, management_ip))
+    logger.info('Restoring snapshot {0}...'.format(snapshot_id))
     client = utils.get_rest_client(management_ip)
     execution = client.snapshots.restore(
         snapshot_id, not without_deployments_envs, force)
-    logger.info('Started workflow\'s execution id: {0}'.format(execution.id))
+    logger.info('Started workflow execution (id: {0})'.format(execution.id))
```
Comment (EarthmanT) [inline cloudify_cli/commands/snapshots.py]: No where above have you put "x process (id: {0})" in the message it's usually like "deployment {0}" so it should be "execution id: {0}"

**Pull request merged**